lib/site_prism/element_container.rb

```diff
@@ -196,8 +180,8 @@ def create_invisibility_waiter(element_name, *find_args)
       method_name = "wait_until_#{element_name}_invisible"
       create_helper_method(method_name, *find_args) do
         define_method(method_name) do |timeout = Capybara.default_max_wait_time, *runtime_args|
-          unless element_does_not_exist?(*self.class.merge_args(find_args, runtime_args, visible: true, wait: timeout))
```

Comment:
Passing the `wait` option in the specific find options allows for use with Capybaras `threadsafe` mode - https://github.com/teamcapybara/capybara#threadsafe